ple/games/pong.py

```diff
@@ -206,18 +217,21 @@ def __init__(self, width=64, height=48, MAX_SCORE=11):
 
     def _handle_player_events(self):
         self.dy = 0
-        for event in pygame.event.get():
-            if event.type == pygame.QUIT:
-                pygame.quit()
-                sys.exit()
 
-            if event.type == pygame.KEYDOWN:
-                key = event.key
-                if key == self.actions['up']:
-                    self.dy -= self.players_speed
+        # consume events
+        pygame.event.get()
+        keys = pygame.key.get_pressed()
 
-                if key == self.actions['down']:
-                    self.dy += self.players_speed
+        if keys[self.actions['up']]:
```

Comment:
Unfortunately this it was not correct to change it. I change it back because the expression is not meant to check if a key is present in the dictionary, but instead we know that it is present there, but we check if its value is set to True, otherwise, this will not work.